TestSwitchApi/TestSwitchApi/Controllers/SessionsController.cs

```diff
@@ -31,7 +31,15 @@ public ActionResult<CandidateTestModel> AddTestSubmission([FromRoute] string tok
             var candidate = _candidates.GetCandidateByGuid(tokenId);
             if (candidate!=null)
             {
-                return _submissions.AddTestSubmission(candidate.Id, newSubmission);
+                try
+                {
+                    _submissions.AddTestSubmission(candidate.Id, newSubmission);
+                    return StatusCode(200);
```

Comment:
This can just be `return Ok()` I think! :) 

But in any case I think I prefer what you had before - was there a reason to change this?

Reply:
I have returned it back to the previous code. I had made these changes to accommodate primary key exceptions. But these exceptions occured only in dev mode . The previous code would have worked in the production env